assets/styles/components/section-titles/_chapters.scss

```diff
@@ -19,6 +19,9 @@
       hyphens: none;
       text-align: $chapter-title-align;
       text-transform: $chapter-title-text-transform;
+      letter-spacing: $chapter-title-letter-spacing;
```

Comment:
Sorry, one more thing—these should all be `if-map-get($chapter-title-letter-spacing, $type)` etc.